cogstat/cogstat.py

```diff
@@ -587,7 +590,8 @@ def filter_outlier(self, var_names=None, mode='2.5mad'):
             If cases were filtered, then filtered and remaining cases are shown.
         """
         mode_names = {'2sd': _('Mean ± 2 SD'),  # Used in the output
-                      '2.5mad': _('Median ± 2.5 MAD')}
+                      '2.5mad': _('Median ± 2.5 MAD'),
+                      'multiple': _('MCCD Mahalanobis distance with .05 chi squared')}
```

Comment:
should we name it 'mahalanobis' instead of 'multiple' to be similarly specific to the single variable methods?

please, add the option to the docstring too

the recommended alpha (0.05?) could be integrated to the method, similar to the SD and MAD methods, where the cutoff cannot be changed here; in this case, the alpha parameter can be removed (but the value can be used in the mode name and in the docstring)

Reply:
Alpha integrated, variable renamed and the docstring was changed accordingly.